docs/userguide/scheduling/resource-propagating.md

```diff
@@ -716,7 +716,9 @@ spec:
     - member3
   #...
   suspension:
-    clusterNames: ["member3"] # Modify the cluster name here
+   dispatchingOnClusters:
+    clusterNames:
+     - member3 # 修改此处的集群名称
```

Comment:
This is the English doc, right?

Reply:
It was copied directly from the Chinese document by mistake.